org.lflang.tests/src/org/lflang/tests/TestBase.java

```diff
@@ -85,7 +84,7 @@ public abstract class TestBase {
     private static final PrintStream err = System.err;
 
     /** Execution timeout enforced for all tests. */
-    private static final long MAX_EXECUTION_TIME_SECONDS = 60;
+    private static final long MAX_EXECUTION_TIME_SECONDS = 300;
```

Comment:
Do we really want to increase the timeout by factor 5? Why is this needed?

Reply:
:man_shrugging: WDYT, @petervdonovan?

Reply:
I myself almost forgot that I did this. Here is the commit message that I wrote last week:
> Try to address failing Python docker tests.
> Warning: increase in TestBase.MAX_EXECUTION_TIME_SECONDS! This is
apparently necessary because of the quantity of dependencies that must
be downloaded in order to run Python Docker tests.

Apparently a test timed out once because of this. We can reset the timeout to what it was before, but there may be occasional test failures as a result that will waste our time.

I suppose the downside of increasing the timeout is that it will waste resources in terms of energy and CI runners if we let lots of tests time out e.g. with livelocks without noticing and cancelling everything. I'm not sure how serious an issue this would be.

Reply:
Do you think `120` or `180` would suffice? I agree with @cmnrd that `300` seems excessive.

Reply:
I do not know, but probably.

Reply:
Timeout reduced to 180 seconds by [6ab91c2](https://github.com/lf-lang/lingua-franca/pull/1221/commits/6ab91c28e61082bd71f5f5de72a779b898e844b1); please unresolve if this is not satisfactory. It takes me 88 seconds (n=1) to run a Python Docker test, so I don't think it's excessive to set the timeout to be double that.